eth-rpc/server.nim

```diff
@@ -35,20 +36,15 @@ const
       (INVALID_REQUEST, "No id specified")
     ]
 
-template ifDebug*(actions: untyped): untyped =
-  # TODO: Replace with nim-chronicle
-  when not defined(release): actions else: discard
+when not defined(release):
```

Comment:
I'm curious about this change. Is there a difference in code generation?

Reply:
+1. Old variant looks cleaner to me

Reply:
With latest devel old variant gives me:
```
server.nim(86, 46) template/generic instantiation from here
server.nim(89, 3) template/generic instantiation from here
server.nim(48, 19) Error: obsolete usage of 'defined', use 'declared' instead
```

Reply:
This surely looks like some bug. Besides, I can't reproduce it with current devel (f99acdb07584d2c4c5eb1f22d998d97bcd823357).

Reply:
Updated my nim to 
```
Nim Compiler Version 0.18.1 [Windows: amd64]
Compiled at 2018-06-08
Copyright (c) 2006-2018 by Andreas Rumpf

git hash: f99acdb07584d2c4c5eb1f22d998d97bcd823357
active boot switches: -d:release
```
And when compiling via `nim c server.nim` i'm still getting error
```
server.nim(86, 46) template/generic instantiation from here
server.nim(89, 3) template/generic instantiation from here
server.nim(48, 19) Error: obsolete usage of 'defined', use 'declared' instead
```

Reply:
Further investigation revealed a supposedly nim bug: https://github.com/nim-lang/Nim/issues/7997. Thus current solution may serve as a workaround for that, i guess.